### DISCUSSION TITLE
Datetime picker would be great addition

### DISCUSSION THREAD

**Discussion by mark1502** [saadeghi/daisyui#1823] [Ideas / feature requests]
In certain spots, a datetime picker is really needed.  But there doesn't appear to be a good tailwind based datetime picker available.  Most of the available options use jquery and bootstrap.  And the html datetime-local option is limited and doesn't work on Firefox.

I'd love to be able to stay within daisyui for this, so I'd like to request adding a datetime picker to the data input components.

Thank you for daisyUI!

**Comment by saadeghi** [comment]
Discussed here:  
- https://github.com/saadeghi/daisyui/discussions/1558
- https://github.com/saadeghi/daisyui/discussions/619
